liveagent-plugin-versions.user.js

```diff
@@ -1,7 +1,7 @@
 // ==UserScript==
 // @name         LiveAgent - Latest plugin versions
 // @namespace    https://theeventscalendar.com/
-// @version      2.22.1.1
+// @version      2.22.2.0
```

Comment:
Third digit is for hotfixes or feature releases.
This release was a maintenance release, so the second digit should be increased.